File: src/com/kostu96/bots/entities/plants/Grass.java
package com.kostu96.bots.entities.plants;

import com.kostu96.bots.World;
import com.kostu96.bots.utils.ImageManager.ImageID;
import com.kostu96.bots.utils.Point;

public class Grass extends Plant {
	private static final int STRENGTH = 0;

	public Grass(World world, Point position) {
		super(world, position, STRENGTH);
	}

	@Override
	public ImageID getImageID() {
		return ImageID.GRASS;
	}

}

File: src/entities/grass.py
from .plant import Plant

class Grass(Plant):
    color = (00, 220, 20)
    symbol = 'GS'

    def __init__(self, world, position):
        super().__init__(world, position, 0)

    def __str__(self):
        return 'Grass'

    def getColor(self):
        return Grass.color

    def getSymbol(self):
        return Grass.symbol
